Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/SliderCombo.cpp

```diff
@@ -256,40 +256,15 @@ SliderDoubleCombo::~SliderDoubleCombo()
 
 void SliderDoubleCombo::setValueSlider(double value)
 {
-    const bool doEmit = m_value != value;
     m_value = value;
-
     updateSpinBox();
-    updateSlider();
```

Comment:
I think this logic was for protecting that we don't double trigger updates, if the user inputs the value from either the slider or the input field, so it doesn't propagate twice. Is that still working as expected?

Reply:
Yes, I've tested this in most of the slider combos and it responds correctly.